src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -9073,6 +9095,13 @@ namespace HVACUnitarySystem {
 		NoLoadHumRatOut = 0.0;
 		OnOffAirFlowRatio = 0.0; //Autodesk:Init Patch to prevent use uninitialized in calls to SimVariableSpeedCoils
 
+		if ( UnitarySystem( UnitarySysNum ).CondenserNodeNum != 0 ) {
+			OutdoorDryBulb = Node( UnitarySystem( UnitarySysNum ).CondenserNodeNum ).Temp;
+		} else {
+			OutdoorDryBulb = OutDryBulbTemp;
+		}
```

Comment:
This block was not there before because the model didn't check MinOAT to see if the compressor could operate. I added this at line 9137 to not bother with testing the output as if the compressor was on when the coil model would model it as off.

    // disable compressor if OAT is below minimum outdoor temperature
    if ( OutdoorDryBulb < UnitarySystem( UnitarySysNum ).MinOATCompressorCooling ) {
        SensibleLoad = false;
        LatentLoad = false;
    }